tests/fail/parse/unexpected-character.fathom

```diff
@@ -0,0 +1,3 @@
+//~ exit-code = 1
+
+🥸
```

Comment:
Note that we really should be testing the error message here, either with some comment-based expectations or golden files. Yet another thing I've not set up, however. 😳